doc/CONFIGURATION.md

```diff
@@ -162,6 +165,10 @@ node:
       natPortMap: "true"
       # enables p2p stream capabilities, equivalent to ipfs p2p
       enableP2PStreams: "true"
+    # enables private libp2p swarm mode
+    # the value here will be used as the private network shared secret key
+    # default is empty string ("") which means use the public network
+    swarm_key: 6f0cf24809cda96d0d68b0fa8abdd64eccd1b4866262040cdc5afecba3418aad
```

Comment:
Should specify the format and size of the key here (32byte, hex). Also, the sample key should be a fake key or not given, otherwise it will get used unintentionally.